src/components/Button.tsx

```diff
@@ -1,98 +1,108 @@
-/* @flow */
-
-import * as React from 'react';
-import { StyleSheet, View, TouchableOpacity, Text } from 'react-native';
-
-import type { Theme } from '../types/Theme';
-import type {
-  TextStyleProp,
-  ViewStyleProp,
-} from 'react-native/Libraries/StyleSheet/StyleSheet';
+import React from 'react';
```

Comment:
```suggestion
import * React from 'react';
```

Also make sure to change other places